atom/browser/web_contents_preferences.cc

```diff
@@ -405,30 +399,24 @@ void WebContentsPreferences::AppendCommandLineSwitches(
 
 void WebContentsPreferences::OverrideWebkitPrefs(
     content::WebPreferences* prefs) {
-  prefs->javascript_enabled =
-      IsEnabled(options::kJavaScript, true /* default_value */);
-  prefs->images_enabled = IsEnabled(options::kImages, true /* default_value */);
-  prefs->text_areas_are_resizable =
-      IsEnabled(options::kTextAreasAreResizable, true /* default_value */);
-  prefs->navigate_on_drag_drop =
-      IsEnabled(options::kNavigateOnDragDrop, false /* default_value */);
+  prefs->javascript_enabled = IsEnabled(options::kJavaScript);
```

Comment:
Aren't the defaults now `false` ?

Reply:
the default values are set in `WebContentsPreferences::SetDefaults()` called from the constructor and the merge method. therefore it should not be necessary to specify the default again value when calling `IsEnabled`

Reply:
ah missed that, in which case the signature of `IsEnabled` can be made stricter, just `bool IsEnabled(const base::StringPiece& name) const;`

Reply:
modified `IsEnabled()` as suggested